addons/web/static/src/webclient/actions/action_service.js

```diff
@@ -451,6 +451,7 @@ export function makeActionManager(env, router = _router) {
                         return stateToUrl(controller.state);
                     },
                     onSelected() {
+                        controller.action.from_breadcrumb = true;
```

Comment:
doesn't seem to be used, and could not find an enterprise PR. Are you sure this is necessary and why ?

Reply:
yep, and we don't want to see that there :-)

Reply:
Thank you for the review. There is a enterprise PR here: https://github.com/odoo/enterprise/pull/69185
Given the specs of the task, it is specifically required to have a particular behavior when we audit an amount in the accounting reports and then come back to the report using the breadcrumb. We want a different behavior if we open the report using the breadcrumb than from opening the report from the menu item. 

spec from the task:
![image](https://github.com/user-attachments/assets/c5390ce5-3541-4b09-aa74-e349f4e91203)

This is the only way I've found to solve this solution, and I think it's actually not bothering at all, as it simply adds a key in the dict of the action. If I'm not mistaken, this dict will be removed right after the action is done anyway. If you really don't want this solution, would you mind suggest another possible approach ?

Reply:
Either you want to implement exactly what has been asked (i.e. restoring the state when coming back with the breadcrumbs), and you should use the `getLocalState` feature from `useSetupAction` which allows to export a state that you want to maintain. This state is given in props when you come back to that action, s.t. you can restore it.

Or you want the last selection to be kept, even if you leave and come back later (e.g. from the menu), like it seems to be the non multi ledger case. In that case, you should use the local storage, which you already do.

Reply:
cth notified labe about this, let's see what she says, but in any cases, you don't need to add anything in the framework

Reply:
Thanks for your reply and for the suggested solution. I've managed to do something that works perfectly, I'll improve tomorrow because it might not be very clean for now. :smiley: